packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -790,8 +791,15 @@ export class ContainerRuntime extends TypedEventEmitter<IContainerRuntimeEvents>
     private latestSummaryGCVersion: GCVersion;
     // This is the source of truth for whether GC is enabled or not.
     private readonly shouldRunGC: boolean;
+    // This is the runtime version when doc is first created
+    private readonly createDocRuntimeVersion: string;
+    // This is the time stamp of when doc is first created
+    private readonly createDocTimeStamp: number;
+    // This is the counter of summary
+    private summaryCount: number;
```

Comment:
Please add these properties after `shouldRunSweep` so that the GC properties are together.

Reply:
Also, I would not use the word `doc` but use `container`.